From: Michael Stapelberg <michael@stapelberg.de>
Date: Thu, 18 Jul 2013 21:09:42 +0200
Subject: Make net.ifnames opt-in instead of opt-out

With this commit, you need to actively specify net.ifnames=1 to get
predictable network interface names:
http://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/
---
 rules/80-net-name-slot.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rules/80-net-name-slot.rules b/rules/80-net-name-slot.rules
index c5f1b38..4738e53 100644
--- a/rules/80-net-name-slot.rules
+++ b/rules/80-net-name-slot.rules
@@ -5,7 +5,7 @@ SUBSYSTEM!="net", GOTO="net_name_slot_end"
 NAME!="", GOTO="net_name_slot_end"
 
 IMPORT{cmdline}="net.ifnames"
-ENV{net.ifnames}=="0", GOTO="net_name_slot_end"
+ENV{net.ifnames}!="1", GOTO="net_name_slot_end"
 
 NAME=="", ENV{ID_NET_NAME_ONBOARD}!="", NAME="$env{ID_NET_NAME_ONBOARD}"
 NAME=="", ENV{ID_NET_NAME_SLOT}!="", NAME="$env{ID_NET_NAME_SLOT}"
